src/p1/section.rs

```diff
@@ -36,18 +36,7 @@ impl Section {
             }
         }
 
-        let body = match &self.body {
-            None => None,
-            Some(body) => match body {
-                _ if body.1.starts_with(r"\/") =>
-                {
-                    #[allow(clippy::single_char_pattern)]
-                    Some((body.0, body.1.strip_prefix(r"\").expect("").to_string()))
-                }
-                _ if body.1.starts_with('/') => None,
-                _ => self.body.clone(),
-            },
-        };
+        let body = self.body_without_comment();
```

Comment:
Why is this needed? Is this not already taken care of in `ignore_comments`? If not so, please do it then. This should be just the `body`. if there's any redundant code, please remove them.

Reply:
body_without_comment() is only used by remove_comments() which in turn is only used by ignore_comments() in the interpreter.